### PR TITLE
Accessibility checks

### PR DESCRIPTION
**Pull request by learnwithalfred**
## This is the accessibility report ( Report Added to PR)

- Page titles - `No issue found.`

- Images text alternatives - `No issues found`

- Headings `No Issues found`

- Contrast - `There was an issue with my bottom background , but i have fixed them` [Fixed Issue](https://github.com/learnwithalfred/portfolio-website/blob/52e0f27ebd9b8bf19086905a5cead047e36ea7ff/style.css#L231
)

- Resize - `There was an issue with page resizing. But it has been fixed` [Fixed Issue](https://github.com/learnwithalfred/portfolio-website/blob/52e0f27ebd9b8bf19086905a5cead047e36ea7ff/style.css#L557
) the same issue has been fixed in several lines of the project.

- Interaction - `There were no labels with the forms. But Labels has been added for screen readers only` [Fixed Issue](https://github.com/learnwithalfred/portfolio-website/blob/52e0f27ebd9b8bf19086905a5cead047e36ea7ff/index.html#L253
)

- Moving content - `No issues found`

- Multimedia - `No issues found`

- The basic structure of the page  - `No Issues found`

### Accessibility checks using Devtools are performed on code (image below).

accersibility-report

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,36 @@
...
+
+- Resize - `There was an issue with page resizing. But it has been fixed`
+
+- Interaction - `There were no labels with the forms. But Labels has been added for screen readers only`
+
+- Moving content - `No issues found`
+
+- Multimedia - `No issues found`
+
+- The basic structure of the page  - `No Issues found`
```
Comment (nicupop729) [inline README.md:31]: Congratulations on your very good work :clap: 

However, this report, as stated in the project requirements, should be in the description of the pull request and not here, in the README file; it is not relevant for a user to read it - users will feel it navigating the app.

Screenshot from project requirements:
image


- [ ] Kindly, remove the report from the README but paste it in the description of the pull request.

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @learnwithalfred & @dqarias, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
 - [X] Correct GitHub flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by learnwithalfred** [comment]
Thanks so much. The issue will be fixed

File: README.md (hunk 1 above)
Comment (learnwithalfred) [reply README.md]: ok
The issues is resolved

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @learnwithalfred, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job in removing the accessibility report from the readme

 ## Required Changes ♻️
- [ ] Kindly add the accessibility report in the PR description [here](https://github.com/learnwithalfred/portfolio-website/pull/9#issue-1224250462).

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by koomekevo — APPROVED** [review]
Hi @learnwithalfred, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**